build/common.ps1

```diff
@@ -189,6 +189,13 @@ Function Install-NuGet {
     & $NuGetExe locals all -list -verbosity detailed
 }
 
+Function Add-ApexFeed {
+    if (Test-Path $NuGetExe) {    
+        # Add Apex VSTS feed
+        & $NuGetExe sources add -Name ApexFeed -Source "https://devdiv.pkgs.visualstudio.com/_packaging/ApexTestFramework/nuget/v3/index.json" -UserName $env:apexUsername -Password $env:apexPat
```

Comment:
change the source url into a variable too.

also use these variables:  VstsPackageFeedUrl , VstsPackageFeedUsername , VstsPackageFeedPassword